packages/core/src/browser/tree/tree-widget.ts

```diff
@@ -382,13 +382,14 @@ export class TreeWidget extends VirtualWidget implements StatefulWidget {
 
     protected renderNode(node: TreeNode, props: NodeProps): h.Child {
         const attributes = this.createNodeAttributes(node, props);
-        return h.div(attributes,
+        const contentEl = h.div({ className: "theia-TreeNodeContent" },
```

Comment:
1. Please use a [constant](https://github.com/theia-ide/theia/blob/c6c0e22176962853201a646ac632e072918c37d9/packages/core/src/browser/tree/tree-widget.ts#L34) for the class name.
 2. It is most likely just me, but `contentEl` means nothing more than `content`.